src/EA.Weee.Web/Views/OrganisationRegistration/TonnageType.cshtml

```diff
@@ -0,0 +1,36 @@
+﻿@using EA.Weee.Web.RazorHelpers
+@using MvcNavigationHelpers
+@model EA.Weee.Web.ViewModels.OrganisationRegistration.Type.TonnageTypeViewModel
+@{
+    ViewBag.Title = "How many tonnes of Electrical or Electronic Equipment (EEE) do you produce a year?";
+}
+
+
+@section full_main_content{
```

Comment:
@will-hope98 I forgot about this (so added). Wrapping the form in @section full_main_content fixes any spacing issues as it adds the form into the correct layout.

Reply:
Ah ok